frontend/packages/kubevirt-plugin/integration-tests-cypress/view/selector.ts

```diff
@@ -47,11 +48,14 @@ export enum nicDialog {
 
 // disk modal
 export enum diskDialog {
-  addDisk = 'button[id="add-disk"]',
   source = 'button[id="disk-select-source"]',
+  sourceURL = 'input[id="disk-url"]',
```

Comment:
nitpick: For sourceURL, container and diskPVC, they're all source type, they can have a unique prefix, like diskURL, diskContainer and diskPVC.

Reply:
Right. I'll fix it